test/test_eos_modifiers.cpp

```diff
@@ -61,34 +117,14 @@ SCENARIO("EOS Builder and Modifiers", "[EOSBuilder][Modifiers][IdealGas]") {
           REQUIRE(unmod.IsType<IdealGas>());
         }
       }
-    }
-    WHEN("We use the EOSBuilder") {
```

Comment:
Unified with the above test case, as EOSBuilder now no longer is a special code path.